### PR TITLE
Support new 'date' and 'time' types in Cassandra

### PR DESCRIPTION
**Pull request by aholmberg**
CASSANDRA-7523
PYTHON-190

### CODE REVIEW

File: cassandra/marshal.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def _make_packer(format_string):
     unpack = lambda s: packer.unpack(s)[0]
     return pack, unpack
 
+uint32_pack, uint32_unpack = _make_packer('I')
```
Comment (aholmberg) [inline cassandra/marshal.py]: Why no explicit big-endian '>' ?

File: cassandra/query.py [hunk 2]
```diff
@@ -498,6 +498,7 @@ def bind(self, values):
                 col_type = col_spec[-1]
 
                 try:
+                    print "About to serialize with col_type: " + str(col_type)
```
Comment (aholmberg) [inline cassandra/query.py]: Leftover debug statement.

**Comment by aholmberg** [comment]
I put a couple of comments on the diff. @josh-mckenzie would you consider rebasing to squash out some of the 'cleanup print'-type commits?
Otherwise I can rebase and tweak myself.

**Comment by josh-mckenzie** [comment]
Sure thing - I'll try to get to that today and push updates.

**Comment by josh-mckenzie** [comment]
Removed debug print, removed redundant uint32_pack that was missing endianness, squashed to single commit, and branch is update.

**Comment by aholmberg** [comment]
@josh-mckenzie I'd like to get an idea for where this C* issue lands in your list of priorities. Are you gunning for next 2.1.x release with this?

**Pull request merged**